File: 설유민/Lv.1/12901-2016년/solution.js
function solution(a, b) {
    let dayWk = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"];
    let days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    return dayWk[(days.slice(0, a - 1).reduce((sum, month)=>{return sum + month;}, 0) + b) % 7];
}

File: 설유민/Lv.1/12901-2016년/solution.py
def solution(a, b):
    day_wk = ['THU', 'FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED']
    days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    return day_wk[(sum(days[:a - 1]) + b) % 7]